example_files/example_lint_cfg.json

```diff
@@ -8,5 +8,6 @@
     "long_lines": { "max_length": 80 },
     "in3": { "indentation_spaces": 4 },
     "continuation_line": { "indentation_spaces": 4 },
-    "in9": { "indentation_spaces": 4 }
+    "in9": { "indentation_spaces": 4 },
+    "in1_common_indentation": 4
```

Comment:
I would've expected this parameter to have a single declaration in this file. Let's have in3, in6 and in9 be declared without any parameters. You can declare in1 with the parameter once.